code/model/UserDefinedForm.php

```diff
@@ -55,6 +57,11 @@ class UserDefinedForm extends Page {
 	 * @var array
 	 */
 	protected $fieldsFromTo = array();
+	
+	/**
+	 * Encryption key to use during encryption and decryption
+	*/
+	protected $key = "d0a7e7997b6d5fcd55f4b5c32611b87cd923e88837b63bf2941ef819dc8ca282";
```

Comment:
The key should be kept out of the source code, and moved into a constant somewhere (like _ss_environment or a yml file). Alternatively, set it as `private static $key`, so that it can be overridden by Config API. Your implementation forces the dev to modify the module directly to change the default key, which makes upgrading more difficult.